spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -58,7 +69,7 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-kafka</artifactId>
-			<version>${spring-integration-kafka.version}</version>
+			<version>2.1.1.BUILD-SNAPSHOT</version>
```

Comment:
This stuff will all be sorted out when boot 1.5.4 is released. I have scheduled 2.1.1 for tomorrow.